src/styles/blocks/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  display: flex;
```

Comment:
The use of `display: flex;` on the `.card` block may not be necessary as it's main purpose is to layout its children. Since the `.card` block itself doesn't have any direct flex children, this property can be removed.